--- /tmp/PreferencesCore_all.js 2016-11-20 12:09:50.346000000 -0200
+++ PreferencesCore_all.js      2016-11-20 12:11:47.295000000 -0200
@@ -2979,29 +2979,6 @@
        buffer[_i++] = data["id"];
        buffer[_i++] = "_PRIMARY_VISIBLE' tabindex=0></span></td></tr>";
         }
-       buffer[_i++] = "<tr><td colspan=2><hr></td></tr><tr><td style='text-align:left' colspan=2><b>";
-       buffer[_i++] = ZmMsg.accountFromPrompt;
-       buffer[_i++] = "</b></td></tr><tr><td class='ZOptionsLabel'>";
-       buffer[_i++] = ZmMsg.fromLabel;
-       buffer[_i++] = "</td><td class='ZOptionsField'>";
-       buffer[_i++] = ZmMsg.fromDetail;
-       buffer[_i++] = "</td></tr><tr><td class='ZOptionsLabel'>&nbsp;</td><td class='ZOptionsField ZOptionsSubField'><table role=\"presentation\"><tr>";
-        if (appCtxt.get(ZmSetting.FROM_DISPLAY_ENABLED)) {
-       buffer[_i++] = "<td style='padding-right:.5em;'><input id='";
-       buffer[_i++] = data["id"];
-       buffer[_i++] = "_PRIMARY_FROM_NAME' tabindex=0 size=30></td>";
-        }
-       buffer[_i++] = "<td><select id='";
-       buffer[_i++] = data["id"];
-       buffer[_i++] = "_PRIMARY_FROM_EMAIL' tabindex=0 size=40></select></td></tr></table></td></tr><tr><td class='ZOptionsLabel'>";
-       buffer[_i++] = ZmMsg.setReplyTo;
-       buffer[_i++] = "</td><td class='ZOptionsField'><input id='";
-       buffer[_i++] = data["id"];
-       buffer[_i++] = "_PRIMARY_REPLY_TO' tabindex=0 type=checkbox></td></tr><tr><td class='ZOptionsLabel'>&nbsp;</td><td class='ZOptionsField ZOptionsSubField'><table role=\"presentation\"><tr><td style='padding-right:.5em;'><input id='";
-       buffer[_i++] = data["id"];
-       buffer[_i++] = "_PRIMARY_REPLY_TO_NAME' tabindex=0 size=30></td><td class='ZmSelector'><select id='";
-       buffer[_i++] = data["id"];
-       buffer[_i++] = "_PRIMARY_REPLY_TO_EMAIL' tabindex=0></select></td></tr></table></td></tr><tr valign=top><td class='ZOptionsLabel'>";
        buffer[_i++] = ZmMsg.signatureLabel;
        buffer[_i++] = "</td><td class='ZOptionsField' valign='middle'>";
        buffer[_i++] = ZmMsg.manageSignatures;
